pm_uep_opt_write/run.sh

```diff
@@ -1 +1,12 @@
-for x in $1/*/*/*.yaml ;do pm-uep-opt $x;done #pm-uep-opt 
\ No newline at end of file
+ex=0
```

Comment:
Can this script be updated/reverted given the changes that Simone made in pymuon-suite?  I don't know exactly what will work but we shouldn't need to grep any more

Reply:
changed to use returned error